config/properties.json

```diff
@@ -1,27 +1 @@
-{
-  "props": {
-    "cname": 16,
-    "props": {
-      "$_dirty": "__d",
-      "$_disable": "__x",
-      "$_listeners": "__l",
-      "$_renderCallbacks": "__h",
-      "$__key": "__k",
-      "$__ref": "__r",
-      "$normalizedNodeName": "__n",
-      "$nextBase": "__b",
-      "$prevContext": "__c",
-      "$prevProps": "__p",
-      "$prevState": "__s",
-      "$_parentComponent": "__u",
-      "$_componentConstructor": "_componentConstructor",
-      "$__html": "__html",
-      "$_component": "_component",
-      "$__preactattr_": "__preactattr_"
-    }
-  },
-  "vars": {
-    "cname": -1,
-    "props": {}
-  }
-}
\ No newline at end of file
+{"props":{"props":{"$_dirty":"__d","$_disable":"__x","$_listeners":"__l","$_renderCallbacks":"__h","$__key":"__k","$__ref":"__r","$normalizedNodeName":"__n","$nextBase":"__b","$prevContext":"__c","$prevProps":"__p","$prevState":"__s","$_parentComponent":"__u","$_componentConstructor":"_componentConstructor","$__html":"__html","$_component":"_component","$__preactattr_":"__preactattr_","$vnode":"i","$debounceRendering":"t","$defaultProps":"l","$afterMount":"o","$componentDidMount":"u","$base":"s","$dangerouslySetInnerHTML":"v","$ref":"h","$render":"p","$componentWillMount":"g","$componentWillReceiveProps":"m","$props":"k","$syncComponentUpdates":"S","$shouldComponentUpdate":"P","$componentWillUpdate":"T","$getChildContext":"_","$componentDidUpdate":"j","$afterUpdate":"C","$beforeUnmount":"H","$componentWillUnmount":"L","$setState":"M","$forceUpdate":"$","$h":"I","$cloneElement":"N","$Component":"O","$rerender":"U","$exports":"q","$preact":"A"}},"vars":{"props":{}}}
```

Comment:
I was confused by the duplicate `props` key, but this file is autogenerated that way

Reply:
do you've any idea about why this doens't keep previous format?

Reply:
Unfortunately not. The build output seems to be unaffected though, as far as I can tell.